.github/workflows/build-and-test.yml

```diff
@@ -8,6 +8,20 @@ on:
   pull_request:
 
 jobs:
+  build:
```

Comment:
Why did we add this job? It doesn't do anything that the `test` job doesn't already do